README.md

```diff
@@ -14,6 +14,7 @@ Check out the [README](/modules/oe_content_entity/README.md) of the module.
 
 - [Requirements](#requirements)
 - [Installation](#installation)
+  - [Upgrade to 1.12.0](#upgrade-to-1120)
```

Comment:
Not sure this will be merged before releasing 1.12.0 since it's not priority for R8.